docs/source/installation.md

```diff
@@ -28,28 +55,113 @@ docker run \
     -v /usr/local/Ascend/driver/lib64/:/usr/local/Ascend/driver/lib64/ \
     -v /usr/local/Ascend/driver/version.info:/usr/local/Ascend/driver/version.info \
     -v /etc/ascend_install.info:/etc/ascend_install.info \
-    -it quay.io/ascend/cann:8.0.rc3.beta1-910b-ubuntu22.04-py3.10 bash
+    -it quay.io/ascend/cann:8.0.0.beta1-910b-ubuntu22.04-py3.10 bash
 ```
 
-You do not need to install `torch` and `torch_npu` manually, they will be automatically installed as `vllm-ascend` dependencies.
+Then you need to install vllm from souce code.
 
-### Manual installation
+```bash
+git clone https://github.com/vllm-project/vllm
+cd vllm
+VLLM_TARGET_DEVICE=empty python setup.py install
+```
 
-Or follow the instructions provided in the [Ascend Installation Guide](https://ascend.github.io/docs/sources/ascend/quick_install.html) to set up the environment.
+Then you can install vllm-ascend pre-built wheel or from source code.
 
-## Building
+### Pre-built wheels (Not support yet)
 
-### Build Python package from source
+```bash
+pip install vllm-ascend -f https://download.pytorch.org/whl/torch/
+```
+
+### Build wheel from source
 
 ```bash
 git clone https://github.com/vllm-project/vllm-ascend.git
 cd vllm-ascend
-pip install -e .
+pip install -e . -f https://download.pytorch.org/whl/torch/
```

Comment:
better to add a note for why we need `-f https://download.pytorch.org/whl/torch/`